modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a single service and apply it to all of your projects. You can configure this type of environment by setting your Ingress Controller to use the `NodePortService` endpoint publishing strategy.
+
+Consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to a node even if the node gets displaced. 
+
+Before you `NodePort`-type `Service` for each project, read the following considerations:
+
+* You must create a wildcard DNS record for the Ingress Controller.
+* You must create a route service and specify the hostname of your Ingress Controller in this service.
+* must preserve the non-default port in the curl in order to access, for example: curl <route>:<port> // what does this mean?
```

Comment:
What does this line mean? Can one do this with a curl command? Curl usually just fetches information?

Reply:
For context - here is what the curl command looks like. You have to use the nodeport of the svc for this ingress controller:
```
[root@openshift-jumpserver-0 ~]# oc get svc/router-nodeport-nodeportsvc -n openshift-ingress
NAME                          TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)                                     AGE
router-nodeport-nodeportsvc   NodePort   172.30.128.82   <none>        80:32136/TCP,443:30999/TCP,1936:31257/TCP   90d
[root@openshift-jumpserver-0 ~]# oc get route
NAME              HOST/PORT                                             PATH   SERVICES          PORT       TERMINATION   WILDCARD
hello-openshift   hello-openshift.nodeportsvc.ipi-cluster.example.com          hello-openshift   8080-tcp                 None
[root@openshift-jumpserver-0 ~]# curl hello-openshift.nodeportsvc.ipi-cluster.example.com:32136
Hello OpenShift!

```

Reply:
Updated to state "You must use the `curl` command to transfer data to the `NodePort`-type `Service` in the Ingress Controller." An example can be in the actual procedure.